Vagrantfile

```diff
@@ -27,8 +26,7 @@ $bootstrap = <<BOOTSTRAP
   sudo chown -R vagrant /var/lib/
   sudo chown -R mysql /var/log/mysql
   sudo chown -R mysql /var/log/mysql
-  sudo apt-fast -y install emacs24-nox
-  sudo apt-fast -y install libmagickwand-dev
+  sudo apt-fast -y install emacs24-nox findutils realpath libmagickwand-dev
```

Comment:
emacs is, strictly speaking, not required for our build. Can we come up with a way to make it optional, so that there's less build time for people who don't want it on their vagrant box?